client/packages/system/src/Name/ListView/utils.ts

```diff
@@ -0,0 +1,27 @@
+import { TypedTFunction, LocaleKey } from '@common/intl';
+import { ArrayUtils, Formatter } from '@common/utils';
+import {
+  ImportRow,
+  LineNumber,
+} from './ImportProperties/PropertiesImportModal';
+
+export const importFacilitiesPropertiesToCsv = (
+  facilities: Partial<ImportRow & LineNumber>[],
+  t: TypedTFunction<LocaleKey>,
+  properties?: string[]
+) => {
+  // TODO maybe don't need the facilities[0] fallback.
```

Comment:
Yeah I would remove... when you call this you're always passing in an array anyway! Also, should this method be called `exportFacilit...`?